piker/ui/_chart.py

```diff
@@ -19,11 +19,11 @@
 
 """
 import time
-from typing import Tuple, Dict, Any, Optional
+from typing import Tuple, Dict, Any, Optional, List
```

Comment:
in 3.9 I'm pretty sure we don't need this?

Reply:
OK, removed in `afafe56` below.